### PR TITLE
fix delete documents by filter documentation

### LINKED ISSUE #535: Documentation on delete documents by filter shows incorrect code
**Description**
Documentation on delete documents by filter shows incorrect code. 
[Link](https://www.meilisearch.com/docs/reference/api/documents#delete-documents-by-filter)

**Expected behavior**
It should the code for delete the documents using a filter

**Current behavior**
Currently, it is showing for getting the documents

**Screenshots or Logs**
Screenshot 2024-05-23 at 8 16 40 PM

**Environment (please complete the following information):**
Not applicable

### PR DESCRIPTION
**Pull request by the-sinner**
# Pull Request

## Related issue
Fixes #535 

## What does this PR do?
- fixes the delete documents by filter documentation 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by ellnix — CHANGES_REQUESTED** [review]
Well spotted! Thanks for taking the time, if you could also add a `=` in the same commit so that we don't spam the commit history too much.

Thanks again :smile:

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ get_documents_post_1: |-
     fields: ['title', 'genres', 'rating', 'language']
   )
 delete_documents_by_filter_1: |-
-  client.index('movies').get_documents(filter: 'genres = action OR genres adventure')
+  client.index('movies').delete_documents(filter: 'genres = action OR genres adventure')
```
[the remark this answers is not in the document]
Comment (the-sinner) [reply .code-samples.meilisearch.yaml:13]: sure, will do the changes

[the remark this answers is not in the document]
Comment (the-sinner) [reply .code-samples.meilisearch.yaml]: done

**Review by ellnix — APPROVED** [review]
LGTM!

**Comment by ellnix** [comment]
bors merge

**Pull request merged**